README.md

```diff
@@ -2,6 +2,11 @@
 
 ## About
 
+ActiveModelSerializers is undergoing some rennovations.
+
+In the mean time, if you're looking for a json:api renderer, [jsonapi-rb](http://jsonapi-rb.org/) will meet your needs --
+with a [performance boost](https://gist.github.com/NullVoxPopuli/748e89ddc1732b42fdf42435d773734a), too.
+
```

Comment:
Maybe instead of adding this text  here, add here a link to ```[Development Status](#status-of-ams)``` and then put this down there?

Reply:
And maybe reference https://github.com/rails-api/active_model_serializers/tree/benchmarks per discussion in https://github.com/rails-api/active_model_serializers/pull/2169 ??